compatibility/bazel_tools/testing.bzl

```diff
@@ -700,6 +700,15 @@ def sdk_platform_test(sdk_version, platform_version):
     # for older versions we still have to disable ClosedWorldIT
     (extra_sandbox_next_args, extra_sandbox_next_exclusions) = (["--implicit-party-allocation=false"], []) if versions.is_at_least("1.2.0", platform_version) else ([], ["--exclude=ClosedWorldIT"])
 
+    if versions.is_at_least("1.17.0", platform_version):
+        extra_sandbox_next_args += ["--max-deduplication-duration=PT5S"]
+
+    # https://github.com/digital-asset/daml/commit/60ffb79fb16b507d4143cfc991da342efea504a7
+    # introduced a KV specific dedup test and we need to disable the non-kv test in return.
+    kv_dedup_version = "1.17.0-snapshot.20210910.7786.0.976ca400"
+    if versions.is_at_least(kv_dedup_version, sdk_version) and versions.is_at_least(kv_dedup_version, platform_version):
```

Comment:
Should these only be excluded if both sdk and platform are more recent or if either is? I.e. should this be an `or`?

Reply:
No for older ledger API test tool versions the two are identical (or rather the kv one does not exist). And for older ledger versions, the KV test suite doesn’t pass while the old one does.

Reply:
:+1: Thanks!